packages/banner/src/Banner.tsx

```diff
@@ -331,7 +104,13 @@ export default function Banner({
       role="alert"
       className={cx(
         baseBannerStyles,
-        bannerVariantStyles[variant].body,
+        bodyTypeScaleStyles[baseFontSize],
+        bannerVariantStyles[theme][variant].body,
+        {
+          [css`
+            padding-right: 36px; // add space for the icon
+          `]: dismissible,
+        },
```

Comment:
Could we break this out as something like `dismissibleBannerStyles` and use a `spacing` token?

Reply:
I can break this out but spacing doesn't have `36`